pkgs/top-level/all-packages.nix

```diff
@@ -8995,6 +8995,8 @@ with pkgs;
 
   kakoune-cr = callPackage ../tools/misc/kakoune-cr { crystal = crystal_1_2; };
 
+  kamp = callPackage ../by-name/ka/kamp/package.nix { };
+
```

Comment:
```suggestion
```

This is unnecessary because the package is in by-name.

Reply:
Thanks, I will change it!